README.md

```diff
@@ -1,20 +1,56 @@
 <p align="center">
-	<img width=256px src="GitHub/logo.png" />
+    <img width=256px src="GitHub/logo.png" />
 </p>
 
 # Dynamic FPS
 
-Dynamic FPS automatically reduces the speed at which Minecraft renders when it's not focused (to 1 FPS) or hidden (no renders at all). It also fixes a bug in Vanilla Minecraft that makes it take much more performance in the background than it should.
+Dynamic FPS can reduce the frame rate, volume, and pause toasts while Minecraft is not focused or hidden.  
```

Comment:
maybe "toast notifications" in case some users don't know what this is?

Reply:
https://minecraft.fandom.com/wiki/Toasts

Reply:
Maybe a good middle ground is just linking to this page?

Reply:
> Maybe a good middle ground is just linking to this page?

yeah, or just write down what it is